bot/seasons/evergreen/space.py

```diff
@@ -0,0 +1,154 @@
+import logging
+from datetime import datetime
+from random import randint
+from typing import Any, Dict, Optional
+from urllib.parse import urlencode
+
+from discord import Embed
+from discord.ext.commands import Cog, Context, command
+
+from bot.bot import SeasonalBot
+from bot.constants import Tokens
+
+logger = logging.getLogger(__name__)
+
+# NASA API base URL
+BASE_URL = "https://api.nasa.gov"
+NASA_IMAGES_BASE = "https://images-api.nasa.gov"
+EPIC_BASE_URL = "https://epic.gsfc.nasa.gov"
+
+# Default Parameters:
+# .apod command default request parameters
+APOD_PARAMS = {
+    "api_key": Tokens.nasa,
+    "hd": True
```

Comment:
Does this just mean that we get a link to the `hd` image or do we receive the actual HD image in data in the response to our request as well? If the latter, do we need that solely to get the "hd" photo link?